README.md

```diff
@@ -57,16 +60,18 @@ The areas that should be implemented are:
 - Testimonial:
   - Each row should link to an individual testimonial. This page does not need to exist, changing the URL is good enough.
 
+Note, that although we believe in agile development, this is a ~~contract job~~ job application, and we will not be available like a normal client would be so to don't expect any regular feedback which is a core part of all agile development methodologies. We also expect your sprints to be no more than a few hours long, with a deadline of a day at most. We'd accept 2 days if you do all the above mind reading well enough.
+
 Other notes:
 
-- All filtering of the testimonials should be done at the server-side level via API calls.
-- You are not expected to implement the functionality of the top-bar, but you may.
-- You are not expected to handle empty or error states, but you may.
-- Consider the audience of Exercism when making choices.
+- All filtering of the testimonials should be done at the server-side level via API calls (we don't expect the Earth!).
+- You are not expected to implement the functionality of the top-bar, but you may (if you do then you will get extra brownie points).
+- You are not expected to handle empty or error states, but you may (we don't handle error states at Exercism!).
+- Consider the audience of Exercism when making choices (not like our judgement on the job market or hiring web developers/~~gettting a job done for free as we're too cheap for Fiverr~~).
 
 ## API Calls
 
-To get data you can use Exercism's API. There are two endpoints, both which return JSON and neither of which need authentication:
+To get data, you can use Exercism's API (which we're sure you're all intimately knowledgable about - who wouldn't be?!). There are two endpoints, both of which return JSON and neither of which need authentication:
```

Comment:
The explanation is right after this sentence. Do you think this wording improves the explanation?